kurtosis-module/impl/module_io/default_params.go

```diff
@@ -43,10 +50,12 @@ func GetDefaultExecuteParams() *ExecuteParams {
 	return &ExecuteParams{
 		Participants: []*ParticipantParams{
 			{
-				ELClientType:  ParticipantELClientType_Geth,
-				ELClientImage: useDefaultElImageKeyword,
-				CLClientType:  ParticipantCLClientType_Nimbus,
-				CLClientImage: useDefaultClImageKeyword,
+				ELClientType:     ParticipantELClientType_Geth,
+				ELClientImage:    useDefaultElImageKeyword,
+				ELClientLogLevel: unspecifiedLogLevel,
+				CLClientType:     ParticipantCLClientType_Nimbus,
+				CLClientLogLevel: unspecifiedLogLevel,
```

Comment:
For consistency, can we keep the same ordering of (type, image, loglevel) so that it will match what's in the README?